alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -56,10 +57,16 @@ Proceed with the following steps to fix the issue:
 1. Use the logs to guide you to the place where the conflicting targets are defined.
 2. Remove the duplicated targets or ensure that distinct targets are labeled uniquely.
 
-### The logs only revolve around the same scrape pool
+### Target is exposing duplicated samples for the same timestamp
```

Comment:
(nit)
```suggestion
### Target exposing duplicated samples for the same timestamp
```

Reply:
https://github.com/openshift/runbooks/pull/210#discussion_r1741965268 :)

Reply:
All good then!

Reply:
(I'll never get why I can't resolve my own comments)

Reply:
weird, I'm able to resolve mine.